packages/flutter_secure_storage/example/tizen/tizen-manifest.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="org.tizen.flutter_secure_storage_tizen_example" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
+    <profile name="common" />
+    <ui-application appid="org.tizen.flutter_secure_storage_tizen_example" exec="Runner.dll" type="dotnet" multiple="false" nodisplay="false" taskmanage="true">
+        <label>flutter_secure_storage_tizen_example</label>
+        <icon>ic_launcher.png</icon>
+        <metadata key="http://tizen.org/metadata/prefer_dotnet_aot" value="true" />
+    </ui-application>
+    <feature name="http://tizen.org/feature/screen.size.all" />
+</manifest>
```

Comment:
Missing newline

Reply:
oh... Tizen extension on vs code doesn't allow last newline. I just found that and I'll fix it.